hopp/resource/wave_resource.py

```diff
@@ -0,0 +1,135 @@
+import pandas as pd
+import PySAM.WaveFileReader as wavefile
+
+from hopp.log import hybrid_logger as logger
+from hopp.resource.resource import *
+
+
+class WaveResource(Resource):
+    """
+    Class to manage Wave Resource data
+    """
+    def __init__(self, lat, lon, year, path_resource="", filepath="", **kwargs):
+        """
+        
```

Comment:
I think it would be really helpful to describe the format of the resource file, what fields are required, the format, etc... and then link to the example file in the test folder

Reply:
Added in https://github.com/NREL/HOPP/pull/133/commits/ea458193cbb6edd0b46cb7b8337a80257ea2f95e